cilium-dbg/cmd/endpoint_get.go

```diff
@@ -55,21 +52,13 @@ var endpointGetCmd = &cobra.Command{
 				os.Exit(1)
 			}
 			return
+		} else {
```

Comment:
Probably we can remove the `command.OutputOption` conditional check as well since both cases are running the same code.

Reply:
Made changes as requested.